docs/source/serving/sky-serve.rst

```diff
@@ -81,21 +126,38 @@ Use :code:`sky serve status` to check the status of the service:
 
 If you see the :code:`STATUS` column becomes :code:`READY`, then the service is ready to accept traffic!
 
-Simply ``curl -L`` the service endpoint --- for the above example, use
-``44.211.131.51:30001`` which automatically load-balances across the two replicas:
+Simply ``curl -L`` the service endpoint, which automatically load-balances across the two replicas:
 
-.. code-block:: console
+.. tab-set::
 
-    $ curl -L <endpoint-url>/generate \
-        -X POST \
-        -d '{"inputs":"What is Deep Learning?","parameters":{"max_new_tokens":20}}' \
-        -H 'Content-Type: application/json'
+    .. tab-item:: vLLM
+        :sync: vllm-tab
 
-    # Example output:
-    {"generated_text":"\n\nDeep learning is a subset of machine learning that uses artificial neural networks to model and solve"}
+        .. code-block:: console
+
+            $ curl -L 3.83.159.229:30001/v1/chat/completions \
+                -X POST \
+                -d '{"model": "mistralai/Mixtral-8x7B-Instruct-v0.1", "messages": [{"role": "user", "content": "Who are you?"}]}' \
```

Comment:
Is the Mixtral model correct? Not vicuna?

Actually, if it's feasible could we replace vicuna with mixtral, since the latter is better quality and more popular?

Reply:
Now changed to mixtral 🫡 PTAL again